frontend/src/components/Hub/ResourceGroup.tsx

```diff
@@ -19,28 +21,17 @@ const ResourceGroup = ({ title, kind }: Props) => {
         templateColumns={["1fr", "1fr", "repeat(2, 1fr)", "repeat(3, 1fr)"]}
         gap="1em"
       >
-        {kind === "video" && (
-          <>
+        {kind === "video" &&
```

Comment:
also enums! 

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Since we are not using documentation type resources, I think im gonna delete it from both the backend and frontend